packages/database/src/migrations/20210719023634-CorrectNumberTeachersByEducationLevel-modifies-data.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+
+import { insertObject, generateId, findSingleRecord, findSingleRecordBySql } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const CODE = 'LESMIS_teachers_education_level_district';
+
+const NEW_REPORT_CONFIG = {
+  fetch: {
+    dataElements: [
+      'noteach_pe_public_f',
+      'noteach_pe_public_m',
+      'noteach_pe_private_f',
+      'noteach_pe_private_m',
+      'noteach_se_public_f',
+      'noteach_se_public_m',
+      'noteach_se_private_f',
+      'noteach_se_private_m',
+    ],
+    aggregations: [
+      {
+        type: 'MOST_RECENT',
+        config: {
+          dataSourceEntityType: 'sub_district',
+        },
+      },
+    ],
+  },
+  transform: [
+    {
+      transform: 'select',
+      "'name'":
+        "translate($row.dataElement, { noteach_pe_public_f: 'Primary Education (Public)', noteach_pe_public_m: 'Primary Education (Public)', noteach_pe_private_f: 'Primary Education (Private)', noteach_pe_private_m: 'Primary Education (Private)', noteach_se_public_f: 'Secondary Education (Public)', noteach_se_public_m: 'Secondary Education (Public)', noteach_se_private_f: 'Secondary Education (Private)', noteach_se_private_m: 'Secondary Education (Private)' })",
+      '...': ['value', 'dataElement'],
+    },
+    'keyValueByDataElementName',
+    {
+      name: 'group',
+      transform: 'aggregate',
+      '...': 'sum',
+    },
```

Comment:
You really don't need to review this all - only the first and third transform steps have changed!

Before it was doing a `"last"` when aggregating and grouping, which is obviously wrong when we want to sum up the levels. We can't sum the org unit and period, but don't actually need them, so the first transform step now drops them from the start.